### ISSUE TITLE
Crimson dependency should be removed

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ontopia/ontopia#273]
```
When I try to use Ontopia together with my project which uses the org.w3c.dom 
interface I receive the following error:

java.lang.AbstractMethodError: 
org.apache.crimson.tree.ElementNode.getTextContent()Ljava/lang/String;

It seems that Ontopia uses an old crimson implementation of that interface 
which no longer implements the interface correctly. I wonder if this dependency 
is still used, since the project seems to be "in hibernation".

Regards,
Christian



What steps will reproduce the problem?
1.
2.
3.

What is the expected output? What do you see instead?


Please use labels and text to provide additional information.

```

Original issue reported on code.google.com by `eadgh...@googlemail.com` on 29 Jun 2010 at 2:00

**Comment by qsiebers** [comment]
As per #481, the first item of [my earlier comment](https://github.com/ontopia/ontopia/issues/273#issuecomment-81979092) can be disregarded

**Comment by qsiebers** [comment]
I've opened a branch (starting at b2b82f7e3c661949b95b12d0ef967dd72ce48da8) to remove Crimson and Xerces as direct dependencies of Ontopia.

**Comment by qsiebers** [comment]
Modifications on the xml-parser-removal branch so far:
 
**Fixed test cases**
 * b2b82f7e3c661949b95b12d0ef967dd72ce48da8 Removed noxlinkns.xtm, as it contains invalid XML
 * c5430dc2fa72cd8f01c691a4547cb33d2afcfbd4 Fixed missing public doctype
 * 0971bfbf5976c1b8ac8bb1a3c78a2656f3d98259 Catching previously hidden exceptions
 * b50eeb2b3bfe9d6e17f4265aade0489848a74caf Fixed jetty interaction

**Code cleanup**
e2fbd0362e4eba73e4ac27399163b175b55dc67e, e9612be4557258bf45e6faf6bedcc2e5f0701031, 0417d0c0b9502cb58f4b3cfcd79580356bb83385, 7913399ae1b5213973000642c134d05a39a6ef01

**Upgraded to Sax2**
4d98d7a90863437ef711672b244b114a20c6e7a9, 86e81cc3ba5ede165ce2d8a03484f17eee6b68c0, b3d335c31cd03cefad73aca7547b2fffa07641fa, eb755f44e442cd237263bee393df4a4129f1ddd1, dc5ad7124e73fcfa2f471f00a6f2c9b4bf047c8c, 745e48443028d8111d0bdda000bbad570495d769, a9b1ed446e35296da7224b365b3199dee2ed0a00

**Removed Crimson and Xerces (and interaction)**
1e870861e2ff386ac2f9b4c259b73d5d4639fb89, 3bb1c59c8246548613087f39000a6be05fbaf456, a6355facd6afa5f310414657822954ac8167a89f

**Upgraded Jena**
a14f008e8ae8812fea6eb225c762db923adc8448, 10fb92f217f4101e261924c506e151b314d18942

**Upgraded POI**
2a289e878376af2b116035b0929d599a40035937, 46a873f01bb921f48545cfd1066b271b3df71a27

**Comment by qsiebers** [comment]
Fixed by 1be9c3fff02748d90a6a08d61fca218b9f2971e8
